wnfs/private/hamt/strategies.rs

```diff
@@ -0,0 +1,165 @@
+use crate::{private::hamt::node::*, BlockStore};
+use anyhow::Result;
+use proptest::{collection::*, prelude::*, strategy::Shuffleable};
+use serde::{de::DeserializeOwned, Serialize};
+use std::{collections::HashMap, fmt::Debug, hash::Hash, rc::Rc};
+
+#[derive(Debug, Clone)]
+pub enum Operation<K, V> {
+    Insert(K, V),
+    Remove(K),
+}
+
+impl<K, V> Operation<K, V> {
```

Comment:
minor: even though gated for tests, it's on by feature, so we should doc these pubs.